### DISCUSSION TITLE
How to calulate the sum of a relationship fields

### DISCUSSION THREAD

**Discussion by Al-Blu** [Budibase/budibase#4510] [Help]
Hi, 
I am just getting started with BudiBase and it is really impressive. 

I have 2 tables for a Company-Survey. One for 'UseCases' and another for 'SuveyResults'. The Use-Case-Table has a relationship to one-to-many SuveyResults. Each SurveyResult has a function-field 'average' with a value. 

So I would like to show the UseCase Table with the following values included:
the count of SuveyResults for each UseCase.
the sum of SuveyResults 'average' divided by the count for each UseCase.

How to I do that?
Thanks !!!!

**Comment by hendrikjap** [comment]
Not supported yet.

**Reply by Al-Blu** [reply]
Thanks for the reply.
